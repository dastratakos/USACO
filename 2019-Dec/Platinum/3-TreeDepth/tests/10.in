50 696 750243917
