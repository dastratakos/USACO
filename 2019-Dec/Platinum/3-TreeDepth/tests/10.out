449512793 49444624 712564120 694762313 239306371 253603336 533190451 398369069 361526878 278308623 574792922 277515625 508820514 723503007 402637769 428971841 387173861 602711957 735429052 173792097 269181379 145740025 392058578 592248870 293782098 625912579 74241665 341952508 388359865 11575395 162281552 234033998 56303077 540027642 106705833 660919683 594355144 318502153 402394567 649181672 716496067 506323399 234681488 599027036 350523714 375398012 14863784 667850178 450716250 361140820
